src/_guides/language/effective-dart/usage.md

```diff
@@ -1105,17 +1100,16 @@ Widget build(BuildContext context) {
   return Row(
     children: [
       RaisedButton(
-        onPressed: increment,
         child: Text('Increment'),
       ),
-      Text('Count: $counter'),
+      Text('Click!'),
     ],
   );
 }
 {% endprettify %}
 
 
-### DON'T use `const` when not needed.
+### DON'T use `const` unnecesarily in code that doesn't need to support Dart 1.
```

Comment:
unnecesarily -> unnecessarily

Reply:
You're right. I was worried about "Effective Dart" prohibiting people from writing code that worked in Dart 1, but I think that ship has sailed. For example, another guideline tells you to use `whereType()`, and that method doesn't work at all in Dart 1 (nor is it even defined in older versions of the SDK).